packages/client/src/utility/general.ts

```diff
@@ -163,3 +163,15 @@ SECRET_KEY="${secretKey ? secretKey : '<your secret key>'}"
 `;
     }
 };
+
+/**
+ * @desc Checks if output and verify if its a JSON object
+ */
+export const isOutputJSON = (output: string) => {
```

Comment:
make output less specific, more of a "give me anything and I'll tell you if it's json viewer compatible"

Reply:
Refactored function to return object if its compatible and null if it isn't.